rdopkg_tar/tar_changes.py

```diff
@@ -123,6 +123,14 @@ def commit_distgit_amend(suffix):
     cmd = ['commit', '-a', '-F', '-', '--amend']
     git(*cmd, input=message, print_output=True)
 
+def check_gitlabuser():
+    # Check for the original commiter username in jenkins env variables.
```

Comment:
I thought your earlier change that used `"""` comments for docstrings was good. Why did you move back to using `#` comments? You can go back to `"""` if you want..